Source/Core/DiscIO/WIABlob.h

```diff
@@ -60,7 +60,7 @@ class WIARVZFileReader final : public BlobReader
   std::string GetCompressionMethod() const override;
   std::optional<int> GetCompressionLevel() const override
   {
-    return static_cast<int>(static_cast<s32>(Common::swap32(m_header_2.compression_level)));
+    return Common::swap32(static_cast<int>(m_header_2.compression_level));
```

Comment:
I don't think it makes sense to have the `static_cast<int>` before the swap32. That function takes and returns a u32.